jalien-root.sh

```diff
@@ -20,14 +20,22 @@ case $ARCHITECTURE in
   osx*) 
 	[[ ! $OPENSSL_ROOT ]] && OPENSSL_ROOT=$(brew --prefix openssl)
 	[[ ! $LIBWEBSOCKETS_ROOT ]] && LIBWEBSOCKETS_ROOT=$(brew --prefix libwebsockets)
+	[[ ! $LIBUV_ROOT && $LIBWEBSOCKETS_ROOT ]] && LIBUV_ROOT=$(brew --prefix libuv)
   ;;
 esac
 
+if [[ "$LIBUV_ROOT" != '' ]]; then
+  export LDFLAGS="$LDFLAGS -L$LIBUV_ROOT/lib -luv"
+  export LD_LIBRARY_PATH="$LIBUV_ROOT/lib:$LD_LIBRARY_PATH"
+  export CXXFLAGS="-I$LIBUV_ROOT/include $CXXFLAGS"
+fi
+
```

Comment:
Why do you need this? Can't cmake detect what to do from `LIBUV_ROOT`?

Reply:
As I said, I was just following other recipes. Are you suggesting just adding `-DLIBUV="$LIBUV_ROOT"` after `-DLWS...`? If so, I just tried it, but cmake doesn't know what to do with it

Reply:
Indeed this is probably something which should be taken into account by JAlien itself.. @Atlantic777 do you actually have the proper find_libuv statements in JAlien-ROOT?

Reply:
Adding @yuw726, he might have more info.